src/gsy_e/gsy_e_core/user_profile_handler.py

```diff
@@ -151,6 +152,18 @@ def _buffer_profile_uuid_list(self):
 
         self._profile_uuids = list(profile_selection)
 
+    @db_session
+    def _buffer_profile_types(self):
+        """ Buffers list of the profile_uuids that correspond to this simulation into
+        self._profile_uuids"""
+        profile_selection = select(
+            (datapoint.profile_uuid, datapoint.profile_type)
+            for datapoint in self.Profile_Database_ConfigurationAreaProfileUuids
+            if datapoint.configuration_uuid == uuid.UUID(gsy_e.constants.CONFIGURATION_ID))
+
+        for p in profile_selection:
+            self._profile_types[p[0]] = InputProfileTypes(p[1])
```

Comment:
This should have raised a pylint warning, maybe your linter was disabled for this file?
```suggestion
        for profile in profile_selection:
            self._profile_types[profile[0]] = InputProfileTypes(profile[1])
```

Reply:
I'll fix this, however it looks like pylint does not complain at all about one-char variables in for loops: 
```
(gsy-e) 192:gsy-e spyrostzavikas$ pylint --version 
pylint 2.11.1
astroid 2.8.4
Python 3.8.9 (default, May 17 2022, 12:55:41) 
[Clang 13.1.6 (clang-1316.0.21.2.5)]
(gsy-e) 192:gsy-e spyrostzavikas$ pylint src/gsy_e/gsy_e_core/user_profile_handler.py 

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

```

Maybe you are using a different pylint version? Because this is the one defined by the project requirements. 
Regardless of whether it is a pylint warning or not, let me fix this.

Reply:
```bash
$ pylint --version
pylint 2.11.1
astroid 2.8.4
Python 3.8.6 (default, Nov  8 2021, 17:05:07)
[Clang 12.0.5 (clang-1205.0.22.9)]
```
It looks like the version is the same, I only have a slightly different python version but I don't think it should matter.